layouts/shortcodes/committers.html

```diff
@@ -0,0 +1,26 @@
+{{ $urlPre := "https://projects.apache.org" }}
+{{ $path :=  "/json/foundation" }}
+{{ $group :=  getJSON $urlPre $path "/groups.json"}}
+{{ $name := getJSON $urlPre $path "/people_name.json" }}
+{{$orgnisations := getJSON "data/teamdata.json" }}
+
+<table class="tableblock frame-all grid-all stretch">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>ID</th>
+      <th>Organisation</th>
+    </tr>
+  </thead>
+  <tbody>
+    {{range $group.camel}}
+    {{ $id:= . }}
```

Comment:
This could also be simpler as
```suggestion
    {{ range $id := $group.camel }}
```